pkg/pipelines/bootstrap.go

```diff
@@ -65,6 +65,11 @@ func Bootstrap(o *BootstrapOptions, appFs afero.Fs) error {
 	}
 	bootstrapped = res.Merge(built, bootstrapped)
 	_, err = yaml.WriteResources(appFs, o.OutputPath, bootstrapped)
+	orgRepo, _ := orgRepoFromURL(o.GitOpsRepoURL)
+
+	if err == nil {
+		log.Successf(fmt.Sprintf("Bootstrapped the %s repo at %s!!", orgRepo, o.OutputPath))
+	}
```

Comment:
A general comment that apples to this PR.   I think logging of successful message is best done in the CLI.

Reply:
Can you explain a bit more about how?

Reply:
Sure.   Logging a brief successful message is a business of the CLI and probably should not be done in the back end..  Here is an example.  https://github.com/dhanashripalodkar/odo/blob/bootstrap/pkg/odo/cli/service/create.go#L245

Reply:
Benefit: You can take the same backend if you wish and frontended it by another CLI or UI that chooses to be more silent or verbose.